### PR TITLE
Added adminAccountsModel

### PR DESCRIPTION
**Pull request by anguyen698**
# Administrative Info
Make sure your branch name conforms to: `feature/hotfix/.../describe_your_branch_with_underscores

### Changes
What changes did you make?
- Added adminAccountsModel

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tested my program thoroughly
- [ ] Did you add/sub fields in the environment file? if so please update .env_sample

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

adminAccountsModel code
admin-account-model

### CODE REVIEW

File: database/model/adminAccountsModel.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const db = require('../dbConfig');
+
+/**
+ * Gets a list of all the admin accounts
+ * @returns {Promise<void>}
+ */
+ async function get () {
+    const admin_accounts = await db('admin_accounts')
+        .get();
```
Comment (YuyusanK) [inline database/model/adminAccountsModel.js:9]: .get() is not a knex function, please take a look at .select() in knex.

**Review by YuyusanK — CHANGES_REQUESTED** [review]
lgtm

**Review by Eldar827 — APPROVED** [review]
lgtm

**Review by YuyusanK — APPROVED** [review]
lgtm

**Pull request merged**